spade-core/src/main/scala/com/salesforce/mce/spade/orchard/OrchardClient.scala

```diff
@@ -40,13 +61,8 @@ class OrchardClient(setting: OrchardClient.Setting) {
           .addQueryParameter("like", pipelineName)
```

Comment:
I think this would need to be change to `s"${pipelineName}#%"` to get all the pipelines in the same group

Reply:
does `like` not detecting `#`? like %pipelineName% not work with %pipelineName#%?